test/csuite/timestamp_abort/main.c

```diff
@@ -395,6 +406,13 @@ thread_run(void *arg)
          */
         if (fprintf(fp, "%" PRIu64 " %" PRIu64 "\n", active_ts, i) < 0)
             testutil_die(EIO, "fprintf");
+
+        if (0) {
+rollback:
```

Comment:
This makes me a bit sad because the test was carefully constructed to have independent key spaces. This makes the code ordering very important. Can you add a comment at the place where we insert into the local table to say that it must appear after the code that uses the transaction. The verification code looks for all-or-none and if that were to move above the transaction the 4 tables could get out of sync.

Reply:
@sueloverso, I've added to the comment as you suggest.